src/components/animated-menu-button/animated-menu-button.js

```diff
@@ -82,7 +82,7 @@ class AnimatedMenuButton extends React.Component {
 
   static defaultProps = {
     direction: 'left',
-    size:      'medium'
+    size: 'medium'
```

Comment:
I want to flag this, on thought, I think there's cases for both ways, depending on whether the values relationships are tabular or key:value. Enforcing one standard over the other might not cover all cases clearly.